### ISSUE TITLE
Improving "project type" dialog

### ISSUE THREAD

**Issue by adriatic** [mattduffield/frontendcreator#26]
At the moment the "new Project" command results with a pop-up with 5 preselected project types:

![image](https://user-images.githubusercontent.com/2712405/37841226-942444f0-2e95-11e8-9a3f-309b1d4f56ec.png)

This can be improved using our experience from Monterey using a wizard which understands the app's supported. I will sketch that and add it to this issue.

**Comment by mattduffield** [comment]
Agreed 100%. This is one aspect that will change over time and a wizard would be a great option. Thanks for helping make this better!

**Comment by adriatic** [comment]
I will create a proposal in the form of a mock-up dialog (I am happily using the moqups.com tool), which will be am improved version of one the Monterey's early concepts:

![image](https://user-images.githubusercontent.com/2712405/37855790-7cc457a0-2ec7-11e8-8f2c-cb51d1971ab9.png)
